ONSdigital/rm-reporting
# Add dashboard eq support

The responses dashboard now has support for eQ surveys, so we need to split our reporting endpoint into `SEFT` and `EQ` as different response statistics are required for each.

### Changes
* Changed responses dashboard resource to have two different paths for eq and seft
* Updated tests
* Fixes url to have dash separated words not underscore separated
* Changed the non CF default database URI to the docker dev postgres local address

### How to test
* Set up and execute both a seft and eq collection exercise
* Get the collection exercise ID's
* Hit the new dashboard eq and seft endpoints with the ID's you've set up
seft: `/reporting-api/v1/response-dashboard/seft/collection-exercise/{id}`
eq: `/reporting-api/v1/response-dashboard/eq/collection-exercise/{id}`

The responses should have different statistics for the seft and eq exercises.

### Links
https://trello.com/c/rdwT6kcO/50-adding-eq-functionality-to-reporting-endpoint
